data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			`	She greets you, "Times are changing quickly, <first>. The Navy sent a message ahead of you. Your information is considerably more detailed, but theirs was enough to anger the entire Elder Council. I advised against rash action, but my words went unheeded."`
+			`	"The Hai Elders are so angered at the killing of our people that we have officially closed our borders. A large armada guards the wormhole at Waypoint. The human authorities have responded by putting a fleet at Ultima Thule. This situation is explosive and may soon lead to war."`
+			`	"Hai are all banned from entering human space now, and humans are not allowed in and out except for critical travel. My new position as an Elder grants me special privileges."`
+			`	"Please, in the name of peace, escort my ship to meet your authorities. I must also see Waypoint and Ultima Thule for myself. I dare not try this without your escort. You will be well-paid."`
+			choice
+				`	"Of course. Let's go."`
+					accept
+				`	"Not yet. I need to make some preparations first."`
+					decline
+	on complete
+		payment 400000
+	npc save accompany
+		government "Hai (Wormhole Access)"
+		personality timid escort
+		ship "Centipede (Teeneep)" "Teeneep's Ship"
+
+mission "Hai Rescue [B02B] Diplomacy: Syndicate (Retry)"
+	landing
+	name "Missing Researchers"
+	description "Visit <waypoints> then escort Teeneep's Centipede to <destination>. Payment is <payment>."
+	source "Hai-home"
+	waypoint "Waypoint"
+	waypoint "Ultima Thule"
+	destination "Sunracer"
+	repeat
+	to offer
+		has "Hai Rescue [B02] Diplomacy: Syndicate: declined"
+		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): failed"
+		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): done"
+		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): active"
+	on visit
+		dialog `To finish this mission, you must visit Waypoint and Ultima Thule, and then land with Teeneep's Centipede.`
+	on enter "Waypoint"
+		dialog `There is a massive fleet of Hai ships here, and more coming in all the time. If this fleet were to enter human space, we might be overrun.`
+	on enter "Ultima Thule"
+		dialog `Although it is reassuring to see humans come together for a common cause, you don't look forward to seeing two species enter an all-out war over a misunderstanding. You also wonder just how effectively the human authorities are keeping the Hai a secret with a huge fleet at an alien station broadcasting Syndicate advertisements...`
+	on enter "Mirfak"
+		dialog `As you enter the system, authorities ask you to land as soon as possible with Teeneep and meet a Syndicate representative.`
+	on offer
+		conversation
+			`Teeneep meets you at your ship again, "Are you ready to escort me to <destination>?"`
+			choice
+				`	"Of course. Let's go."`
+					accept
+				`	"Not yet. I need to make some preparations first."`
+					defer
+	on complete
+		# Lower payment because she's annoyed at the delay.
```

Comment:
On investigation, I'm not sure this is actually possible. It would require placing a branch _before_ the first line of dialogue (of which there is only one). And I don't believe the game allows that. 
Which means such a variation would require an expansion of the text, which I feel is unnecessary.